measures/create_typical_building_from_model/measure.rb

```diff
@@ -417,8 +421,12 @@ def run(model, runner, user_arguments)
       standard.model_modify_infiltration_coefficients(model, primary_bldg_type, climate_zone)
 
       # set ground temperatures from DOE prototype buildings
-      standard.model_add_ground_temperatures(model, primary_bldg_type, climate_zone)
-
+      if standard.respond_to?('model_add_ground_temperatures')
+        standard.model_add_ground_temperatures(model, primary_bldg_type, climate_zone)
+      else
+        # 0.6.0+, model_set_ground_temperatures
+        OpenstudioStandards::Weather.model_set_ground_temperatures(model, climate_zone: climate_zone)
+      end
```

Comment:
same stuff, that was causing an actual crash in model_articulation1 test. Because unlike openstudio_results, which just skips sections/table when something fails (a begin/rescue block is used), this crashes.